src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -43,76 +57,104 @@ To forward your Kubernetes logs to New Relic with our plugin:
 
 New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
```

Comment:
@nbaenam do we need to present this as an alternative to the New Relic plugin? What happens if you use both?